### PR TITLE
Feat(eos_cli_config_gen): Support for EVPN Multihoming IP Mass-withdrawal

### LINKED ISSUE #3545: Support for EVPN Multihoming IP Mass-withdrawal
### Enhancement summary

Support for this new feature described in this TOI: https://www.arista.com/en/support/toi/eos-4-30-1f/17514-evpn-multihoming-ip-mass-withdrawal

Configuration is at the evpn address familty level:
```
   address-family evpn
      route export ethernet-segment ip mass-withdraw 
      route import ethernet-segment ip mass-withdraw
```

Also listed in the tech-library as recommended configuration: 
https://tech-library.arista.com/data_center/evpnvxlan/deployment_guide/domain_b/#evpn-all-active-leaf

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

Higher convergence for EVPN A/A deployments (note not compatible with EVPN DC GW solution or EVPN multihoming single-active).

### Describe the solution you would like

Just additional support for these two commands on eos_cli_config_gen.

### Describe alternatives you have considered

_No response_

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by laxmikantchintakindi**
## Change Summary

Support for EVPN Multihoming IP Mass-withdrawal

## Related Issue(s)

Fixes #3545 

## Component(s) name

`arista.avd.eos_cli_config_gen

## Proposed changes
Added two inputs for under address-family evpn. Update the template  for the new feature.
Added two new keys in address-family evpn/routes in router-bgp schema.

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by Shivani-gslab** [comment]
Please remove the changes inherited from your previous PR. (Create the branch from `devel` branch)

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-evpn-vpn-import-pruning.cfg [hunk 1]
```diff
@@ -62,6 +62,8 @@ router bgp 65101
       neighbor EVPN-OVERLAY-PEERS activate
       no neighbor MLAG-IPv4-UNDERLAY-PEER activate
       route import match-failure action discard
+      route import ethernet-segment ip mass-withdraw
+      route export ethernet-segment ip mass-withdraw
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-evpn-vpn-import-pruning.cfg]: These changes do not match your yaml changes. Please rerun molecule on your side and commit the changes.

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-evpn-vpn-import-pruning.cfg]: I executed molecule again. Now these lines are not present in documents and configs.

**Review by ClausHolbechArista — APPROVED** [review]

**Review by MaheshGSLAB — APPROVED** [review]
LGTM

**Review by gmuloc — APPROVED** [review]

**Pull request merged**